docker/nginx/nginx.dockerfile

```diff
@@ -3,6 +3,8 @@ FROM nginx:latest
 ENV WORKER_PROCESSES auto
 ENV PROXY_PASS http://tfb-database:5000
 
+ADD https://github.com/aspnet/Benchmarks/raw/master/src/Benchmarks/testCert.crt /etc/ssl/certs/testCert.crt
```

Comment:
Shouldn't this be pulling from the local file system?

Reply:
It's on the filesystem if it means it came from github ... It has never failed for me, unless when GH was down, but in this case this file dockerfile can't even be cloned.

But otherwise if it's already there, I agree there is no reason to pull it again. Unless it's in a folder that docker can't access. We do set a docker context after all, that might limit it.

Maybe we should copy these certs in the nginx folder. It would be redundant, but that might make things easier too.

So many options.

Reply:
What are the boundaries for locally accessing files in the repo from inside of a dockerfile? If it can access the repo root folder, I'd suggest to create a new 'certificates' folder there storing the certs. Otherwise, I'd prefer to download them from GitHub, unless there is a strong objection to it.

Reply:
Isn't the docker file just used to build the container? I'd expect it to have full access to the local disk, unless that container is being deployed to another machine?

Reply:
I would expect the 'container builder' to have the full access to the local filesystem, but Sebastian mentioned above that it might not be so.